<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>length</class>
 <widget class="QDialog" name="length">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgba(222, 226, 58, 1);
background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="convert_button">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>600</y>
     <width>231</width>
     <height>99</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Roboto&quot;;
color: rgb(255, 255, 255);
background-color: rgba(97, 144, 198, 1);
font-weight:bold;
border:none;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>Convert</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>300</y>
     <width>94</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Roboto&quot;;
color: rgba(29, 49, 72, 1);
font-weight:bold;</string>
   </property>
   <property name="text">
    <string>From :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="mainwindow_button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>41</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/resource/res/Group 29.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>470</y>
     <width>53</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Roboto&quot;;
color: rgba(29, 49, 72, 1);
font-weight:bold;</string>
   </property>
   <property name="text">
    <string>To :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>471</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 30pt &quot;Roboto&quot;;
font-weight:bold;
color:rgba(97, 144, 198, 1);
</string>
   </property>
   <property name="text">
    <string>CONVERT LENGTH</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="answer_label">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>420</y>
     <width>321</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
color:rgba(97, 144, 198, 1);
font: 75 40pt &quot;Roboto&quot;;
font-weight:bold;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="from_combobox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>290</y>
     <width>440</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
font: 75 15pt &quot;Roboto&quot;;
color: rgba(97, 144, 198, 1);
border:3px solid rgba(29, 49, 72, 1);
border-radius:10px;
padding-left:20px;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="to_combobox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>460</y>
     <width>440</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;
font: 75 15pt &quot;Roboto&quot;;
color: rgba(97, 144, 198, 1);
border:3px solid rgba(29, 49, 72, 1);
border-radius:10px;
padding-left:20px;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="from_value">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>290</y>
     <width>321</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 15pt &quot;Roboto&quot;;
color: rgba(29, 49, 72, 1);
border:none;
border: 3px solid rgba(97, 144, 198, 1);
border-radius:10px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>210</y>
     <width>101</width>
     <height>39</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Roboto&quot;;
color: rgba(29, 49, 72, 1);
font-weight:bold;</string>
   </property>
   <property name="text">
    <string>Value :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_button">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>600</y>
     <width>231</width>
     <height>99</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;Roboto&quot;;
color: rgb(255, 255, 255);
background-color: rgba(97, 144, 198, 1);
font-weight:bold;
border:none;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>390</y>
     <width>511</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/resource/res/GettyImages-455313485-5899349a3df78caebc0c30da-removebg-preview_2-removebg-preview 1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>-120</y>
     <width>511</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/resource/res/GettyImages-455313485-5899349a3df78caebc0c30da-removebg-preview_2-removebg-preview 1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_6</zorder>
  <zorder>label_4</zorder>
  <zorder>convert_button</zorder>
  <zorder>label_2</zorder>
  <zorder>mainwindow_button</zorder>
  <zorder>label_3</zorder>
  <zorder>label</zorder>
  <zorder>answer_label</zorder>
  <zorder>from_combobox</zorder>
  <zorder>to_combobox</zorder>
  <zorder>from_value</zorder>
  <zorder>label_5</zorder>
  <zorder>clear_button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
